environment-approvals.yml

```
@@ -222,6 +222,7 @@ prod:
   - repo: https://github.com/hmcts/family-api-gateway.git
   - repo: https://github.com/hmcts/et-sya-frontend.git
   - repo: https://github.com/hmcts/et-sya-api.git
+  - repo: https://github.com/hmcts/et-hearings-api.git
```

This is for prod approval, usually taken when the infra is approved for prod / ready for OAT
```suggestion
  - repo: https://github.com/hmcts/et-hearings-api.git
```